### PR TITLE
Create de-DE.yml

### PR DESCRIPTION
**Pull request by LeoWinterDE**
#349

Now it is right, I think.

Greetings

### CODE REVIEW

File: languages/de-DE.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        zero: "keine Kategorie gefunden"
+        one: "1 Kategorie gefunden"
+        other: "{n} Kategorien gefunden"
+    tags_found:
+        zero: "keinen Tags gefunden"
+        one: "1 Tag gefunden"
+        other: "{n} Tags gefunden"
+
+pagination:
+    page: "seite %d"
```
Comment (darekkay) [inline languages/de-DE.yml]: `Seite`, as a substantive, has to be capitalized

File: languages/de-DE.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    of: "von %d"
+    newer_posts: "NEUERE BEITRÄGE"
+    older_posts: "ÄLTERE BEITRÄGE"
+    previous: "FRÜHER"
+    next: "NÄCHSTER"
+
+post:
+    no_title : "kein title"
```
Comment (darekkay) [inline languages/de-DE.yml]: → "kein Titel"

**Comment by LouisBarranqueiro** [comment]
@darekkay thanks for your help!
@LeoWinterDE Could you update your PR?

**Comment by LeoWinterDE** [comment]
#398

**Pull request closed without merging**